Library/test-helpers/lib.sh

```diff
@@ -569,10 +570,15 @@ Returns 0 when the start was successful, non-zero otherwise.
 
 limeStartAgent() {
 
-    # print TPM details
-    echo -e "--- TPM details ---"
-    tpm2_getcap properties-fixed | grep -Pzo '(MANUFACTURER|VENDOR_STRING|FIRMWARE_VERSION).*?\n.*?raw.*?\n(.*?value.*?\n)?'
-    echo -e "-------------------"
+    # save TPM details
+    date >> ${__INTERNAL_limeTPMDetails}
+    echo -e "\n# tpm2_getcap properties-fixed" >> ${__INTERNAL_limeTPMDetails}
+    tpm2_getcap properties-fixed >> ${__INTERNAL_limeTPMDetails}
+    echo -e "\n# tpm2_getcap algorithms" >> ${__INTERNAL_limeTPMDetails}
+    tpm2_getcap algorithms >> ${__INTERNAL_limeTPMDetails}
+    echo -e "\n# tpm2_pcrread" >> ${__INTERNAL_limeTPMDetails}
```

Comment:
Perhaps instad of pcrread, we could use `tpm2_getcap pcrs`, for a more compact output? It will  show an output like this, which does not have the measurements we get with pcrread:

```
$ tpm2_getcap pcrs
selected-pcrs:
  - sha1: [ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23 ]
  - sha256: [ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23 ]
```

Reply:
OK